bundles/org.openhab.binding.caddx/README.md

```diff
@@ -0,0 +1,247 @@
+| zone_entry_exit_delay_2                          | Switch    | Configuration       | Entry / exit delay 2                       |
+| zone_interior                                    | Switch    | Configuration       | Interior                                   |
+| zone_local_only                                  | Switch    | Configuration       | Local only                                 |
+| zone_keypad_sounder                              | Switch    | Configuration       | Keypad Sounder                             |
+| zone_yelping_siren                               | Switch    | Configuration       | Yelping siren                              |
+| zone_steady_siren                                | Switch    | Configuration       | Steady siren                               |
+| zone_chime                                       | Switch    | Configuration       | Chime                                      |
+| zone_bypassable                                  | Switch    | Configuration       | Bypassable                                 |
+| zone_group_bypassable                            | Switch    | Configuration       | Group bypassable                           |
+| zone_force_armable                               | Switch    | Configuration       | Force armable                              |
+| zone_entry_guard                                 | Switch    | Configuration       | Entry guard                                |
+| zone_fast_loop_response                          | Switch    | Configuration       | Fast loop response                         |
+| zone_double_eol_tamper                           | Switch    | Configuration       | Double EOL tamper                          |
+| zone_type_trouble                                | Switch    | Configuration       | Trouble                                    |
+| zone_cross_zone                                  | Switch    | Configuration       | Cross zone                                 |
+| zone_dialer_delay                                | Switch    | Configuration       | Dialer delay                               |
+| zone_swinger_shutdown                            | Switch    | Configuration       | Swinger shutdown                           |
+| zone_restorable                                  | Switch    | Configuration       | Restorable                                 |
+| zone_listen_in                                   | Switch    | Configuration       | Listen in                                  |
+| zone_faulted                                     | Switch    | Zone Condition      | Faulted (or delayed trip)                  |
+| zone_tampered                                    | Switch    | Zone Condition      | Tampered                                   |
+| zone_trouble                                     | Switch    | Zone Condition      | Trouble                                    |
```

Comment:
shouldn't this be a Contact instead of a Switch?

Reply:
Yes. Changed.